trustable_cli/metrics.py

```diff
@@ -140,13 +140,24 @@ def get_message_size_metrics(self):
         }
         return metrics
 
-    def get_commits_week_mean(self, days_interval: int):
+    def get_commit_frequency_metrics(self, days_interval: int):
         """
-        Get the average (mean) number of commits per week
+        Get the average (mean) number of commits per week, month and
+        year if the days of interval is greater than the metrics interval.
 
         :param days_interval: Interval of days to calculate the mean
         """
-        return self.total_commits / days_interval / 7
+        metrics = {}
+
+        metrics["week"] = self.total_commits / (days_interval / 7)
+
+        if days_interval >= 30:
+            metrics["month"] = self.total_commits / (days_interval / 30)
+
+        if days_interval >= 365:
+            metrics["year"] = self.total_commits / (days_interval / 365)
```

Comment:
In the cases where this metric can't be calculated, return a None or similar, so the metric is always available.
Also, add a test for those cases.